.github/workflows/tests.yml

```diff
@@ -40,6 +40,13 @@ jobs:
         shell: bash -l {0}
         run: |
           pytest -v tests
+      - name: Test CLI
+        shell: bash -l {0}
+        run: |
+          chemprop train -i tests/data/regression/mol.csv --task-type regression --epochs 1 --num-workers 4
+          chemprop predict --checkpoint tests/data/example_model_v2.ckpt -i tests/data/regression/mol.csv --num-workers 4
+          chemprop train -i tests/data/regression/mol+mol.csv --task-type regression --smiles-columns smiles solvent --epochs 1 --num-workers 4
+          chemprop predict --checkpoint tests/data/example_model_v2_regression_multi.ckpt -i tests/data/regression/mol+mol.csv --smiles-columns smiles solvent --num-workers 4
```

Comment:
this is not how we should be testing the CI. It should be included in the integration tests under pytest and should be part of a separate PR

Reply:
Thanks! I removed it here and opened an issue https://github.com/chemprop/chemprop/issues/590 to keep track of this.